js/myparcel.js

```diff
@@ -786,14 +820,60 @@ MyParcel = {
         return object;
     },
 
+
+    /*
+    * showCutoffTimeCounter
+    *
+    * Count down the hours, minutes and seconds until the point of the cutoff time is reached.
+    *
+    */
+    showCutoffTimeCounter: function () {
+
+        var cutoffTime = MyParcel.data.config.cutoffTime.split(':');
+        var start = new Date;
+
+        /* if when seconds are not entered, it starts counting from 00 */
+        if (cutoffTime[2] === undefined){
+            cutoffTime[2] = '00';
+        }
+
+        /* Place the hours, minutes and seconds separately */
+        start.setHours(cutoffTime[0], cutoffTime[1], cutoffTime[2]);
+
+        function pad(num) {
+            return ("0" + parseInt(num)).substr(-2);
+        }
+
+        /* countdown of the time */
+        function countDown() {
```

Comment:
function in een function? `showCutoffTimeCounter()` is te groot, als je `function countDown()` buiten zet, is het goed